conbench/templates/index.html

```diff
@@ -98,20 +113,12 @@ <h3>Language-independent Continuous Benchmarking (CB) Framework</h3>
 {% block scripts %}
 {{super()}}
 <script type="text/javascript">
-  var table = $('#runs').dataTable( {
-      "responsive": true,
+  var table = $('#runs').DataTable( {
       "order": [[0, 'desc']],
-      "columnDefs": [
-          {
-              "targets": [4],
-              "visible": false,
-              "searchable": true,
```

Comment:
Why can this safely go? For example, what did `"searchable": true,` mean and why is it ok to remove this? :)

Reply:
Previously I was running into a issue described here :https://github.com/conbench/conbench/issues/478#issuecomment-1340697361

I was ignoring the JS bit and wasn't able to find the reason. This points to the col "Hash" to be silent for some reason (which honestly I don't have a solid reasoning to)

Reply:
Ah, this `"targets": [4],` means that the entire option block only targeted one specific column?  OK!

OK. I think this patch is fine then even without further understanding.

Wven if this breaks someone's flow. We have to go through this kind of pain to understand what users want, need.